bin/jwtwebhook.py

```diff
@@ -55,33 +56,14 @@ def handle_request(self):
                         return
                 else:
                     post_body = encoded_post_body
-
-                # Handle JSON
-                try:
-                    parsed_body = flatten(post_body)  # flatten(body_json)
-                except ValueError:
-                    # Could not parse output
-                    parsed_body = None
-
-                    if self.server.logger is not None:
-                        self.server.logger.warn("Content body could not be parsed as JSON")
-
-                # Include the data if we got some
-                if parsed_body is not None:
-                    result.update(parsed_body)
-
-            # Add the data regarding the query
-            result['command'] = self.command
-            result['client_address'] = self.client_address[0]
-            result['client_port'] = self.client_address[1]
```

Comment:
We're throwing away information about where the hook data comes from... do we want to include this differently now?

Reply:
Would need to inject that in the json payload to keep the json raw form aspect. Thoughts?

Reply:
Host or source field perhaps?

Reply:
Host = Client IP address
Source = Splunk HF host